src/Native/SymSgdNative/CMakeLists.txt

```diff
@@ -4,16 +4,18 @@ set(SOURCES
     SymSgdNative.cpp
 )
 
+SET(MKL_LIB_ROOT ${CMAKE_SOURCE_DIR}/../../packages/mlnetmkldeps/${MKL_VERSION}/runtimes)
```

Comment:
I think it would make sense for this whole path to be passed into CMake.  That way we don't need to hard code `${CMAKE_SOURCE_DIR}/../../packages`